exporter/opentelemetry-exporter-prometheus-remote-write/src/opentelemetry/exporter/prometheus_remote_write/gen/remote.proto

```diff
@@ -0,0 +1,82 @@
+// Copyright 2016 Prometheus Team
```

Comment:
it looks like these are copied from the prometheus library itself. Probably fine. 

Are there any thoughts around what the update story would look like? just copy the protos over I suppose.

Reply:
>  just copy the protos over I suppose.
Yup! thats the plan currently